cmd/exchange_template/exchange_template.go

```diff
@@ -108,8 +108,8 @@ func main() {
 	}
 
 	fmt.Println("GoCryptoTrader: Exchange templating tool service complete")
+	fmt.Println("Add appropriate exchange config settings, particularly enabled and available assets and pairs, to config_example.json for proper functionality (it will automatically be added to testdata/configtest.json)")
```

Comment:
I don't think "will be" is correct here. I think it already *has been*.
```suggestion
	fmt.Println("Add appropriate exchange config settings, particularly enabled and available assets and pairs, to config_example.json for proper functionality (it has been automatically be added to testdata/configtest.json)")
```